app/controllers/concerns/api/v2/taxonomies_controller_extension.rb

```diff
@@ -0,0 +1,7 @@
+module Api::V2::TaxonomiesControllerExtension
+  extend Apipie::DSL::Concern
+
+  update_api(:index) do
+    add_scoped_search_description_for(controller_name.classify.constantize)
```

Comment:
can we move this to the taxonomy_controller concern instead of adding another concern for this, similar to what we did in https://github.com/theforeman/foreman/pull/6892?